src/database/migrations/1639431417577-ReportTable.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  MigrationInterface, QueryRunner, Table, TableForeignKey,
+} from 'typeorm';
+
+export class ReportTable1639431417577 implements MigrationInterface {
+  private table = 'reports';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: this.table,
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'created_at',
```

Comment:
move `created_At` an `updated_at` to end, since order of columns in Postgresql is static